packages/components/menu/src/MenuItem/MenuItem.tsx

```diff
@@ -33,7 +33,7 @@ function _MenuItem<E extends React.ElementType = typeof MENU_ITEM_DEFAULT_TAG>(
 ) {
   const { testId, className, as, isInitiallyFocused, ...otherProps } = props;
 
-  const id = useId(null, 'menu-item');
+  const id = useId(undefined, 'menu-item');
```

Comment:
What are these changes about? Is there a type problem?

Reply:
yes, for some reason the build for the website was failing because of the types mismatch. Although I have no idea why it started failing only on this particular PR.